fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -74,6 +78,7 @@ public FiatPermissionEvaluator(Registry registry,
         .build();
 
     this.getPermissionCounterId = registry.createId("fiat.getPermission");
+    this.legacyFallback = configProps.isLegacyFallback();
```

Comment:
worth adding a dynamic props feature flag to this maybe

Reply:
That's reasonable, I'll get that in.